core/src/main/java/com/novoda/noplayer/metadata/Metadata.java

```diff
@@ -0,0 +1,56 @@
+package com.novoda.noplayer.metadata;
+
+import java.util.List;
+
+public class Metadata {
+
+    private final List<Entry> entries;
```

Comment:
cannot we have two entries, one for `BinaryFrame` and another one for `EventMessage`? I'm asking because from the client point of view you are going to force them to inspect the `Entry` interface in order to find out the to what they need to cast to

Reply:
Good point 👍

Reply:
Talked with @laithnurie offline. Looks like the client needs a list of events **inorder**, so we are going to add some javadoc on the types returned